release/_bom.sh

```diff
@@ -146,13 +176,24 @@ function generate_pom_release_bom {
 		> /tmp/artifact_urls.txt
 
 	for artifact_file in $(
-		find "${_BUNDLES_DIR}/osgi" "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF" -name '*.jar' | \
-			sed \
-				-e 's/\.jar$//' \
-				-e "s@.*/@@" \
-				-e "s@-@.@g" | \
-			grep -v -E "(\.demo|\.sample\.|\.templates\.)" | \
-			sort
+		if [[ "${_PRODUCT_VERSION}" == "7.3."* ]]
+		then
+			find "${_BUNDLES_DIR}/osgi" "${_BUNDLES_DIR}/tomcat-9.0.83/webapps/ROOT/WEB-INF" "${_BUNDLES_DIR}/tomcat-9.0.83/lib/ext" -name '*.jar' | \
+				sed \
+					-e 's/\.jar$//' \
+					-e "s@.*/@@" \
+					-e "s@-@.@g" | \
+				grep -v -E "(\.demo|\.sample\.|\.templates\.)" | \
+				sort
+		else
+			find "${_BUNDLES_DIR}/osgi" "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF" -name '*.jar' | \
+				sed \
+					-e 's/\.jar$//' \
+					-e "s@.*/@@" \
+					-e "s@-@.@g" | \
+				grep -v -E "(\.demo|\.sample\.|\.templates\.)" | \
+				sort
```

Comment:
@lucasmiranda0, can you try to reuse this code?

As fair as I can see, the unique difference here is just the directories used by `find`